File: Java/Assignment.java
public class Assignment {
    public static void main (String[] args){
        double right = 6.0;
        double left = 10.0;
        double answer, a, b;

        System.out.println("A = " + left);
        System.out.println("B = " + right);

        // Assignment
        System.out.println("\nAssignment");
        answer = left + right;
        System.out.println("A(" + left +
                           ") + B(" + right +
                           ") = " + answer);

        // Unary Operators
        System.out.println("\nUnary Operator");
        a = left;
        b = right;

        System.out.println("A++ = " + a++);
        a = left;
        System.out.println("++A = " + ++a);
        System.out.println("B-- = " + b--);
        b = right;
        System.out.println("--B = " + --b);

        // Power function
        System.out.println("\nPower Function");
        a = left;
        b = right;

        System.out.println("A^B = " + Math.pow(a, b));
    }
} 

File: C/assignment.c
#include <stdio.h>
#include <math.h>

/*
 * Demonstrate C assignments
 *
 * Compile with: gcc -o assignment assignment.c -lm
 *
 */


int main(void){
	double right = 6.0;
	double left = 10.0;
	double answer, a, b;

	printf("A = %f\n", left);
	printf("B = %f\n", right);
	
	/* Assignment */
    printf("\nAssignment\n");
	answer = left + right;
	printf("A(%f) + B(%f) = %f\n", left, right, answer);
	
	/* Unary Operators */
    printf("\nUnary Operators\n");
	a = left;
	b = right;
	printf("A++ = %f\n", a++);
	a = left;
	printf("++A = %f\n", ++a);
	printf("B++ = %f\n", b++);
	b = right;
	printf("++B = %f\n", ++b);
	
	/* Power function */
    printf("\nPower function\n");
	answer = pow(left, right);
	printf("A ^ B = %f\n", answer);
	
	return(0);
}
